pkg/provider/provider.go

```diff
@@ -220,7 +222,7 @@ func buildTestEnvironment() { //nolint:funlen
 	logrus.Infof("Operators found: %d", len(env.Operators))
 }
 
-func getPodContainers(aPod *corev1.Pod) (containerList []*Container) {
+func getPodContainers(aPod *corev1.Pod, ignoreContainers bool) (containerList []*Container) {
```

Comment:
Suggestion: `ignoreContainers` seems a weird name to me for a param of a function called `getPodContainers`. What about just passing a map of black-listed names to filter out?
```
func getPodContainers(aPod *corev1.Pod, blackListedNames map[string]bool) (containerList []*Container) {
    	for j := 0; j < len(aPod.Spec.Containers); j++ {
		    cut := &(aPod.Spec.Containers[j])
		    if blackListedNames[cut.Name] {
		        continue
		    }
		    
		    container := Container{Podname: aPod.Name, Namespace: aPod.Namespace,
		    ...
		    ...
```

Reply:
Good idea, I wonder if I can pass that list through.  I'll try it.

Reply:
I think changing the name to something like `useIgnoreList` might be more useful.